### PR TITLE
Add tk.deat.Jazz2Resurrection

### PR DESCRIPTION
**Pull request by The-King-of-Toasters**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
	- `--device=all` Used to access gamepads.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/deathkiller/jazz2-native/issues/41
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

Cc @deathkiller

### CODE REVIEW

**Comment by The-King-of-Toasters** [comment]
bot, build tk.deat.Jazz2Resurrection

File: tk.deat.Jazz2Resurrection.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      - --disable-examples
+      - --disable-openmpt123
+      - --disable-static
+      - --disable-tests
+      - --without-portaudio
+      - --without-portaudiocpp
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.so
```
Comment (bbhtt) [inline tk.deat.Jazz2Resurrection.yml:25]:
```suggestion
      - "*.la"
      - "*.so"
```

**Comment by bbhtt** [comment]
Fix the appstream errors and add `developer_name`, create a patch if you want but please submit it upstream.

```json
{
    "errors": [
        "appstream-failed-validation"
    ],
    "warnings": [
        "appstream-missing-developer-name"
    ],
    "appstream": [
        "style-invalid         : <caption> cannot end in '.' [Screenshot of the story selection menu.]",
        "style-invalid         : <caption> cannot end in '.' [Screenshot of Jazz in the level \"Marinated Rabbit\".]",
        "style-invalid         : <caption> cannot end in '.' [Screenshot of Spaz in the level \"Dungeon Dilemma\".]"
    ]
}
```

File: tk.deat.Jazz2Resurrection.yml (hunk 1 above)
Comment (The-King-of-Toasters) [reply tk.deat.Jazz2Resurrection.yml]: FYI I cribbed these lines from other repos that use openmpt, might be worth modifying them too.

**Comment by The-King-of-Toasters** [comment]
bot, build tk.deat.Jazz2Resurrection

**Comment by The-King-of-Toasters** [comment]
bot, build tk.deat.Jazz2Resurrection

**Comment by The-King-of-Toasters** [comment]
Not sure who I'm supposed to poke to get things moving along, but I can confirm everything works great on my Steam Deck (gamescope/plasma X11) and Arch Desktop (sway/plasma wayland). Touchscreen support works too when in Steam Deck desktop mode.

**Comment by barthalion** [comment]
bot, build tk.deat.Jazz2Resurrection

**Pull request closed without merging**